scripts/install.sh

```diff
@@ -55,6 +55,7 @@ install_docker() {
 
     log "⏳ Installing Docker..."
     curl -fsSL https://get.docker.com | sh
+    echo '{"log-driver": "local"}' > /etc/docker/daemon.json
```

Comment:
Please see my other comment about setting the default for each container.

This log-driver configuration will work fine for new setups. But there could be cases when a user wants to preinstall and configure Docker on their own and only then use the server as an uncloud machine. In this case, it's probably not a great idea to mangle their config (it could be owned by a configuration management tool such as Ansible).